packages/cli-kit/src/error.ts

```diff
@@ -75,7 +76,7 @@ export async function handler(error: unknown): Promise<unknown> {
     }
   }
 
-  await outputError(fatal)
+  renderFatalError(fatal)
```

Comment:
Why having both renderError and renderFatalError? Couldn't we have just one and show the stacktrace only when required?

Reply:
I see, thanks! You can forget about this 👍